test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -776,6 +776,39 @@ def forward(self, x):
             model, inputs, f, dynamic_axes={"x": [0, 1]}, input_names=["x"]
         )
 
+    def test_col2im(self):
+        # This test can be moved to test/onnx/test_pytorch_onnx_onnxruntime.py when ORT implement ::Col2Im
+
+        # Random batched RGB 32x32 image-shaped input tensor of batch size 64
+        original_image_inputs = torch.randn((64, 3, 32, 32))
```

Comment:
I think it'd better we set a range [0, 255] if you are simulating image here?

Reply:
Can you elaborate the rationale?
The image size is not that important, we just need one image with known result.
I didnt use same values for for dilation, padding and stride (instead of `1` for everything) to make sure to result would not be a false positive

Reply:
I was referring that pixel range is [0, 255], not the image size.

Reply:
I still don't get it. What would be the code after your proposal? That might help me get your point